src/NIST_800-53/NIST_800-53.ts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
These checks  should be added here
https://github.com/cdklabs/cdk-nag/blob/main/src/NIST-800-53/nist-800-53.ts